.travis.yml

```diff
@@ -1,32 +1,32 @@
-language: generic
+language: node_js
+node_js:
+  - "10"
 
 env:
   - OCAML=4.02.3
   - OCAML=4.02.3+buckle-master
+  - OCAML=4.02.3+buckle-master ESY_BUILD=YES ESY__CACHE=/home/travis/.esy
   - OCAML=4.03.0
   - OCAML=4.04.2
   - OCAML=4.05.0
   - OCAML=4.06.1
   - OCAML=4.07.0
 
-before_script:
-  - sudo add-apt-repository -y ppa:robert7/tidy-html5
-  - sudo apt-get update
-  - sudo apt-get install tidy
-  - wget https://github.com/ocaml/opam/releases/download/2.0.0/opam-2.0.0-x86_64-linux
-  - sudo mv opam-2.0.0-x86_64-linux /usr/local/bin/opam
-  - sudo chmod a+x /usr/local/bin/opam
-  - "opam init -y --compiler=$OCAML --disable-sandboxing"
+cache:
+  directories:
+  - /home/travis/.opam
+  - /home/travis/.esy
+  - /home/travis/.nvm
+  - ./_build/default
```

Comment:
Is this desirable? Do we want to keep cache even for tests executions across builds?

Reply:
I had the same question. You only want to cache builds of your dependencies typically (the `~/.esy/` directory.

Reply:
Coming from `bazel` I'm used to letting the build tool cache very aggressively — assuming we trust `dune` to re-build and re-test incrementally, should this be a problem?

Reply:
I would be fine with this if we cleaned the html `_scratch` directory between test executions. Otherwise the previously generated files that are not generated in next runs might interfere with the result (i.e., it should be an error in this case).